catalog/spatial/geocoding/spatial-geocoding-plugin/pom.xml

```diff
@@ -43,6 +45,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
```

Comment:
outside of scope, but i'm not sure this needs to be listed as a dependency anymore